### PR TITLE
added type error handling and show command doc if error occurred

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: versions.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [versions]
-crate_server = 0.47.7
+crate_server = 0.48.7
```
Comment (chaudum) [inline versions.cfg]: use `0.49.2`

File: versions.cfg [hunk 2]
```diff
@@ -10,7 +10,6 @@ collective.recipe.omelette = 0.16
 crate = 0.13.0
 crate-docs-theme = 0.3.8
 docutils = 0.12
-gp.recipe.tox = 0.3
```
Comment (chaudum) [inline versions.cfg]: do not remove!

File: src/crate/crash/command.py [hunk 3]
```diff
@@ -409,9 +409,19 @@ def _try_exec_cmd(self, line):
...
-            message = cmd(*words[1:])
-            if message:
-                self.logger.info(message)
+            try:
+                message = cmd(*words[1:])
+            except TypeError, typeErr:
+                self.logger.critical(typeErr.message)
+                doc = cmd.__doc__
+                if doc and not doc.isspace():
+                    self.logger.info('help: {0}'.format(words[0].lower()))
```
Comment (chaudum) [inline src/crate/crash/command.py]: "**H**elp: ..."

File: src/crate/crash/command.py [hunk 4]
```diff
@@ -382,8 +382,8 @@ def _switch_format(self, fmt=None):
         return u'{0} is not a valid output format.\nUse one of: {1}'.format(fmt, ', '.join(self.OUTPUT_FORMATS))
 
     def _connect(self, server):
-        """ connect to the given server """
-        self.connection = connect(servers=server, error_trace=self.error_trace)
+        """ connect to the given server, e.g.: \connect localhost:4200 """
+        self.connection = connect(servers=server, timeout=2, error_trace=self.error_trace)
```
Comment (seut) [inline src/crate/crash/command.py]: pls document default timeout change at changes. is this really used at crate-python?

**Comment by seut** [comment]
otherwise ack

**Comment by seut** [comment]
ack

**Pull request merged**